Podfile

```diff
@@ -0,0 +1,18 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '6.0'
```

Comment:
I did small research, and cocoa pods uses platform of the project so that this line is not needed.